Edit/better docs

samirkouider opened the pull request:
Schematics for GCP and Gaggia New Classic have been updated to current nano spec
Guide now fully recommends bypassing the eco board
Formatting - Added warnings
Added o ring to BOM
Moved things around on the stm docs

samirkouider commented:
@Zer0-bit yoooo, you ok reviewing this or you wanting me to change something?

tclift commented:
@samirkouider FYI I'm following your branch for a 1st time GCP build, and making docs changes as I go. I had started similar changes and rebased onto your branch when I saw that you were further along.

samirkouider commented:
> @samirkouider FYI I'm following your branch for a 1st time GCP build, and making docs changes as I go. I had started similar changes and rebased onto your branch when I saw that you were further along.

@tclift Hey man, I've now added complete nano schematics for the GCP instructions. Essentially one could complete the install just by looking at them now. 

Slowly but surely updates will come for other things like PCB and a better STM integration. 

Let me know what specific things you want to see and I can update
